installer/scripts/bootstrap.bash

```diff
@@ -1,30 +1,43 @@
 #!/bin/bash
 
```

Comment:
Why not check ubuntu version and use that to determine the ros name. Then you don't need an argument and you can use the old source+wget structure.

Reply:
That's a good idea. I'll add that.